docs/wiki/research/PBS/ePBS.md

```diff
@@ -0,0 +1,670 @@
+# Enshrined Proposer-Builder Separation (ePBS)
+
+## TLDR
+
+Enshrined Proposer-Builder Separation (ePBS) refers to integrating the PBS mechanism directly into the Ethereum blockchain protocol itself, rather than having it operate through external services or addons. This integration aims to formalize and standardize the separation between the roles of block proposers and block builders within the core protocol rules, enhancing the system's efficiency, security, and decentralization.
+
+
+## What is PBS
+
+Proposer-Builder Separation (PBS) is a design philosophy[^1] and mechanism in within the context of Ethereum, that aims to decouple the roles of proposing blocks (proposers) and constructing the content of those blocks (builders). This separation addresses various challenges and inefficiencies associated with block production, and in the context of maximizing extractable value (MEV). You can learn more about PBS [at here](pbs.md).
```

Comment:
Cross reference and link where mentioned for PBS, MEV and MEVboost.

Reply:
I don't think we need to link pages at every occurrence